lms/static/js/learner_dashboard/views/program_header_view_2017.js

```diff
@@ -0,0 +1,57 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'text!../../../templates/learner_dashboard/program_header_view_2017.underscore',
+        'text!/static/images/programs/micromasters-program-details.svg',
+        'text!/static/images/programs/xseries-program-details.svg',
+        'text!/static/images/programs/professional-certificate-program-details.svg'
+    ],
+         function(Backbone, $, HtmlUtils, pageTpl, MicroMastersLogo,
+                  XSeriesLogo, ProfessionalCertificateLogo) {
+             return Backbone.View.extend({
+                 breakpoints: {
+                     min: {
+                         medium: '768px',
+                         large: '1180px'
+                     }
+                 },
+
+                 el: '.js-program-header',
+
+                 tpl: HtmlUtils.template(pageTpl),
+
+                 initialize: function() {
+                     this.render();
+                 },
+
+                 getLogo: function() {
+                     var logo = false,
+                         type = this.model.get('programData').type;
+
+                     if (type === 'MicroMasters') {
```

Comment:
@AlasdairSwan good idea. The discovery API is currently capable of returning a link to an image for each program type inline with the serialized programs, and I could see us returning a link to an SVG served by discovery/S3. Right now there are PNGs logos for the program types stored in the discovery service.

Let's table this for now, but I (and @clintonb) like the idea of using the logos from the discovery service where possible instead of duplicating them in other services.